nf_core/__main__.py

```diff
@@ -2188,7 +2188,7 @@ def command_create(ctx, name, description, author, version, force, outdir, templ
     log.warning(
         "The `[magenta]nf-core create[/]` command is deprecated. Use `[magenta]nf-core pipelines create[/]` instead."
     )
-    pipelines_create(ctx, name, description, author, version, force, outdir, template_yaml, plain, organisation)
+    pipelines_create(ctx, name, description, author, force, outdir, template_yaml, organisation)
```

Comment:
The `version` argument shouldn't be removed :) 
```suggestion
    pipelines_create(ctx, name, description, author, version, force, outdir, template_yaml, organisation)
```